### ISSUE TITLE
Disable some of the icons

### ISSUE THREAD

**Issue by Quipex** [mallowigi/a-file-icon-idea#152]
Hello, is it possible to disable some of the icons?
If not, it would be cool if such a feature existed.

**Comment by mallowigi** [comment]
Hi, I have this planned but for now I'm taking a little break on this project. 

Thanks

**Comment by yokunjon** [comment]
> Hello, is it possible to disable some of the icons?
> If not, it would be cool if such a feature existed.

Hey! I experienced something similar and solved it by adding custom file association to a non-existent icon.
![image](https://user-images.githubusercontent.com/1013994/113998097-680f1280-9861-11eb-9cc6-f49b13169360.png)

**Comment by Quipex** [comment]
Yeah, it kinda works but there are issues like java.io.FileNotFoundException at IDE Fatal Errors window and it doesn't work for the custom folder icons.
The problem with the custom folder icons is the packages are displayed in full name instead of relative to the parent package 
![image](https://user-images.githubusercontent.com/30311108/114000056-4f076100-9863-11eb-943d-9fa65339c576.png)
